turbinia/task_manager.py

```diff
@@ -49,7 +49,17 @@
 PSQ_QUEUE_WAIT_SECONDS = 2
 
 # Define metrics
-SERVER_TASKS = Gauge('server_tasks', 'Turbinia Server Total Tasks')
+turbinia_server_tasks_total = Gauge(
+    'turbinia_server_tasks_total', 'Turbinia Server Total Tasks')
+turbinia_server_tasks_completed_total = Gauge(
+    'turbinia_server_tasks_completed_total',
+    'Total number of completed server tasks')
+turbinia_jobs_created_total = Gauge(
+    'turbinia_jobs_created_total', 'Total number jobs created')
+turbinia_jobs_resolved_total = Gauge(
+    'turbinia_jobs_resolved_total', 'Total number jobs resolved')
```

Comment:
Do we want to use "completed" instead of "resolved" here to keep it similar to the tasks metrics?  Also, do we want "server" in the name for the same reason?

Reply:
Done